style.css

```diff
@@ -0,0 +1,64 @@
+/* * {
+    margin: 0;
+    padding: 0;
+} */
```

Comment:
I would like to advise you to consider no left commented unused code so you can have a nice and clean code.